README.md

```diff
@@ -34,6 +34,12 @@ This example plugin comes with:
 
 These examples show how to add to Foreman in various ways.
 
+## Foreman plugin labeling
+With the following line in the ruby gemspec file the gem is marked, that it's a foreman plugin. 
+This is especially necessary if you are using React / JS stuff in webpack directory.
+
+    s.metadata    = { "is_foreman_plugin" => "true" }
```

Comment:
Should this use Ruby syntax highlighting?
````suggestion
```ruby
    s.metadata    = { "is_foreman_plugin" => "true" }
```
````